### PR TITLE
feat: Added new BaseFee param inside transactions

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
A short summary which serves as a squashed-commit message.

A description to understand introduced changes without reading the code.

Important changes:
- [x] Something worth noting for reviewers.

Closes #

### CODE REVIEW

File: services/wallet/transfer/commands.go [hunk 1]
```diff
@@ -148,10 +149,11 @@ type controlCommand struct {
 	feed               *event.Feed
 	errorsCount        int
 	nonArchivalRPCNode bool
+	rpcClient          *rpc.Client
```
Comment (alaibe) [inline services/wallet/transfer/commands.go]: I think you don't need to pass rpcClient all the way down, you already should have access to the chainClient in the downloader which is a wrapper around the rpc Client.

File: services/wallet/transfer/downloader.go [hunk 2]
```diff
@@ -162,6 +170,17 @@ func (d *ETHDownloader) getTransfersInBlock(ctx context.Context, blk *types.Bloc
...
+				var feeHistory FeeHistory
+				err = rpcClient.Call(&feeHistory, d.chainClient.ChainID, "eth_feeHistory", "0x1", (*hexutil.Big)(blk.Number()), nil)
+				if err != nil {
+					return nil, err
+				}
+
+				var baseGasFee string = ""
+				if len(feeHistory.BaseFeePerGas) > 0 {
+					baseGasFee = feeHistory.BaseFeePerGas[0]
+				}
```
Comment (alaibe) [inline services/wallet/transfer/downloader.go]: Could you extract this block in the chainClient that will avoid to duplicate it later, wdyt?

File: appdatabase/migrations/sql/1662589108_add_base_fee_transaction.up.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE transfers ADD COLUMN base_gas_fee TEXT NOT NULL DEFAULT "";
+UPDATE transfers SET base_gas_fee = "";
+
+ALTER TABLE pending_transactions ADD COLUMN base_gas_fee TEXT NOT NULL DEFAULT "";
+UPDATE pending_transactions SET base_gas_fee = "";
```
Comment (alaibe) [inline appdatabase/migrations/sql/1662589108_add_base_fee_transaction.up.sql]: You are not updating pending_transactions right?

**Review by alaibe — COMMENTED** [review]
I think what you did will works but the downloader contains everything already, no need to pass down another type.

File: appdatabase/migrations/sql/1662589108_add_base_fee_transaction.up.sql (hunk 3 above)
Comment (Khushboo-dev-cpp) [reply appdatabase/migrations/sql/1662589108_add_base_fee_transaction.up.sql]: damn yes, I will remove that.

File: services/wallet/transfer/downloader.go (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply services/wallet/transfer/downloader.go]: I am not sure I follow @alaibe

File: services/wallet/transfer/commands.go (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply services/wallet/transfer/commands.go]: ok I am trying to use the chainClient now

File: services/wallet/transfer/downloader.go (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply services/wallet/transfer/downloader.go]: ok I got it, you are suggesting we move this piece if code over to the chain client?

**Review by alaibe — APPROVED** [review]

**Pull request merged**